common/model-views.cpp

```diff
@@ -6362,6 +6362,16 @@ namespace rs2
                 ImGui::PopStyleColor();
                 ImGui::PopFont();
             }
+            else if(dev.supports(RS2_CAMERA_INFO_PRODUCT_ID))
+            {
+                std::string device_pid = dev.get_info(RS2_CAMERA_INFO_PRODUCT_ID);
+                if(device_pid == "abcd")// Specific for D457
+                {
+                    ss.str( "" );
+                    ss << "GMSL ";
```

Comment:
Why do we need to print an extra space at the end of the string?

Reply:
done.